languages/wikipron/tsv/por_bz_phonetic.tsv

```diff
@@ -0,0 +1,366 @@
+abafador	a b a f a d o ɹ
+abafador	- d o ɾ
+abafador	a b a f a d o ɻ
+abafador	- d o ɾ
+abatedouro	a b a t e d o w ɾ u
+abatedouro	a b a t e d o w ɾ o
+aberto	a b ɛ χ t u
+aberto	a b ɛ ɹ t u
+abundar	a b ũ d a χ
+abundar	a b ũ d a ɹ
+abundar	a b ũ d a ɻ
+acabamento	a k a b a m ẽ t ʊ
+ácaro	a k a ɾ ʊ
+ácaro	a k a ɾ o
+acender	a s ẽ d e ɾ
+acender	a s ẽ d e ɹ
+acender	a s ẽ d e ɾ
+acender	a s ẽ d e ɻ
+ácido	a s i d u
+aclimatação	a k l i m a t a s ɐ̃ w
+acontecer	a k õ t e s e χ
+acontecer	a k õ t e s e ɾ
+acontecer	a k õ t e s e ɹ
+acontecer	a k õ t e s e ɾ
+acontecer	a k õ t e s e ɻ
+acontecer	a k õ t e s e ɾ
+acontecer	a k õ t e s e χ
+actínio	a k t ʃ i n i u
+adaga	a d a ɡ a
+adenina	a d e n i n a
+adeus	a d e w s
+adição	a d͡ ʒ i s ɐ̃ w
```

Comment:
Oops, we forgot about having to correctly handle the tie bar for affricates. I'm opening an issue for this now.